bin/bugsnag-maze-runner

```diff
@@ -165,79 +68,18 @@ class MazeRunnerEntry
   def start(args)
     # Parse args, processing any Maze Runner specific options
     @args = args.dup
-    options = @parser.parse args
+    options = Maze::OptionParser.parse args
+    return unless options
```

Comment:
If this is just for the help/version cases then I think it's easier to `exit` there; this makes it seem like not getting any options is expected

Reply:
Changed as suggested.

Reply:
Sorry, I meant exit in the rescues in `option_parser`; then you always get `options` here

Reply:
Got you, yes that would be cleaner.